projects/components/src/menu-dropdown/menu-dropdown.component.ts

```diff
@@ -37,4 +40,12 @@ export class MenuDropdownComponent {
 
   @ContentChildren(MenuItemComponent)
   public items?: QueryList<MenuItemComponent>;
+
+  public hasItems$: Observable<boolean> = of(false);
```

Comment:
we can mark it protected.

Reply:
ah I didn't realize that was supported now. will do that going forwarded.